### DISCUSSION TITLE
How do I query a materialized view using sea orm?

### DISCUSSION THREAD

**Discussion by vamshiaruru-virgodesigns** [SeaQL/sea-orm#1244] [Q&A] [answered]
Hi all, beginner to Sea ORM here. In Python world I could query a materialized view using sqlalchemy just as I would query any other table. Is it possible to do the same with sea orm? My materialised view doesn't have any primary key, so how would I define a `DerivedModel` attribute for it? Or do I do this in some other way?
Any help is appreciated, thanks.

**Comment by vamshiaruru-virgodesigns** [comment, marked as answer]
Kind people on discord have answered this question. You basically create just another entity with [DeriveModelEntity] and just provide a dummy primary key. You can now use that entity in querying.
If this is the wrong way to do it, someone can please correct me.

**Reply by billy1624** [reply]
Thanks for the updates!! @vamshiaruru-virgodesigns
And a big shoutout to dragonn#7642 for answering the question on Discord
